src/client/containers/EventsContainer.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+import PropTypes from 'prop-types'
+
+import Grid from '@material-ui/core/Grid'
+import { withStyles } from '@material-ui/core/styles'
+
+import EventsView from '../pages/Events/EventsView'
+
+/*
+*
+* FAKE API HANDLING PART - START
+*/
+// Fake Events API
+import { EventsResponse, EventField, ProvidedTimes } from './EventsFakeAPI'
+
+function getEventsAsArray(givenTime) {
```

Comment:
If you look closely at this function, this will not actually do anything in production. This is essentially a unit test written in production code. So good job with the set up, however this is the wrong place for it